data/sql/updates/pending_db_world/rev_1488880633534999000.sql

```diff
@@ -0,0 +1,32 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1488880633534999000');
```

Comment:
that's not your fix, give credits if you copy paste stuff from https://github.com/dalaranwow/dalaran-wow (https://github.com/dalaranwow/dalaran-wow/issues/4523)